snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedOWLAxiomHelper.java

```diff
@@ -35,7 +35,7 @@ public static String getDefinitionStatusFromExpressions(Set<String> owlExpressio
 		
 		return owlExpressions.stream()
 				.filter(expression -> !Strings.isNullOrEmpty(expression))
```

Comment:
Filter GCI axioms here, not in the create or update requests. They don't have a say in the concept's suggested definition status, but should be going through when creating or updating a SNOMED CT concept, just as before.

Or did they cause some other problem?